tests/functional/test_xdd_load_file.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+#
+# Description - test xdd seek load
+#
+# Validate the functionality of xdd seek load is able to load a file with or without
+# numreqs. It should also overwrite the blocksize and reqsize if provided.
+#
+# Get absolute path to script
+SCRIPT=${BASH_SOURCE[0]}
+SCRIPTPATH=$(dirname "${SCRIPT}")
+
+# Source the test configuration environment
+source "${SCRIPTPATH}/../test_config"
+source "${SCRIPTPATH}/../common.sh"
+
+# Perform pre-test
+initialize_test
+test_dir="${XDDTEST_LOCAL_MOUNT}/${TESTNAME}"
+
+# Function to load a file with optional numreqs, blocksize, and reqsize
+seek_load_file() {
+  local load_file=$1
+  local numreqs=$2
+  local blocksize=$3
+  local reqsize=$4
+  local cmd="${XDDTEST_XDD_EXE} -seek load ${load_file} -target /dev/null"
+
+  # If the optional parameters are provided, add them to the command
+  if [[ -n "${numreqs}" ]]; then
+    cmd+=" -numreqs ${numreqs}"
+  fi
+  if [[ -n "${blocksize}" ]]; then
+    cmd+=" -blocksize ${blocksize}"
+  fi
+  if [[ -n "${reqsize}" ]]; then
+    cmd+=" -reqsize ${reqsize}"
+  fi
+
+  # Execute the command
+  eval "${cmd}"
+}
+
+# Function to verify the output
+verify_output() {
+  local output=$1
+  local expected_blocksize=$2
+  local expected_reqsize=$3
+  local expected_numreqs=$4
```

Comment:
Reorder arguments so that the call looks the same as `seek_load_file`

Reply:
Are 2 functions needed?

Reply:
One function `seek_load_file` is running the program ensures it runs, another function `verify_output` is used to make sure it has expected output.

For example, if I have `-bs 2048` for `-seek load`, it should not override the `blocksize` in the file

Reply:
Ok.